pyiron_gpl/pacemaker/pacemaker.py

```diff
@@ -0,0 +1,231 @@
+# coding: utf-8
+# Copyright (c) ICAMS, Ruhr University Bochum, 2021
+# Distributed under the terms of "GPLv3", see the LICENSE file.
+
+import logging
+import numpy as np
+import os
+import pandas as pd
+import re
+import ruamel.yaml as yaml
+from pyiron_base import GenericJob, GenericParameters
+from pyiron_base.settings.generic import Settings
+from shutil import copyfile
+
+
+s = Settings()
+
+try:
+    from pyace import BBasisConfiguration, ACEBBasisSet
+
+    HAS_PYACE = True
+except ImportError as e:
+    print("Could not import `pyace` package. The package should be installed for proper functionality")
```

Comment:
Please use the [`ImportAlarm`](https://github.com/pyiron/pyiron_base/blob/3295218d87dc1518d6073d9916a859b4e424c4be/pyiron_base/generic/util.py#L238). In your case I guess the decorator would then belong down with `get_{current/final}_potential` and tightens up a smidgen of code duplication.

Reply:
Done